NitroxClient/GameLogic/Spawning/Metadata/Processor/SeamothMetadataProcessor.cs

```diff
@@ -50,29 +46,9 @@ private void SetLights(SeaMoth seamoth, bool lightsOn)
             return;
         }
 
-        // In the future, maybe do a patch for all ToggleLights.SetLightsActive() to handle all cases of lights toggling (i.e. powercell being inserted, etc.)
         using (FMODSystem.SuppressSendingSounds())
```

Comment:
Didn't catch it on mobile but wouldn't we need a `using (PacketSuppressor<NitroxModel.Packets.ToggleLights>.Suppress())` here too?

Reply:
That's a great question; to be honest, all I really did with this PR was undo the changes I made in https://github.com/SubnauticaNitrox/Nitrox/pull/2354 for the SeamothMetadataProcessor and then removed similar looking code in ToggleLightsProcessor (which was the code I copied from for the seamoth code).

I am not sure what the purpose of that suppressor is in ToggleLightsProcessor, so I can't answer that question with any confidence. But if not having it theoretically causes some kind of packet echoing, then it's not something I noticed during IG testing. Should we look more into this?